src/graph/tree/GraphAccountTreeItem.ts

```diff
@@ -17,8 +17,8 @@ export class GraphAccountTreeItem extends DocDBAccountTreeItemBase {
     public static contextValue: string = "cosmosDBGraphAccount";
     public contextValue: string = GraphAccountTreeItem.contextValue;
 
-    constructor(parent: AzureParentTreeItem, id: string, label: string, documentEndpoint: string, private _gremlinEndpoint: IGremlinEndpoint | undefined, masterKey: string, isEmulator: boolean) {
-        super(parent, id, label, documentEndpoint, masterKey, isEmulator);
+    constructor(parent: AzureParentTreeItem, id: string, name: string, label: string, documentEndpoint: string, private _gremlinEndpoint: IGremlinEndpoint | undefined, masterKey: string, isEmulator: boolean) {
```

Comment:
instead of adding a `name: string` parameter, please add a parameter at the end that is `, databaseAccount?: DatabaseAccount` which is only passed in from `CosmosDBAccountProvider` (not AttachedAccountsTreeItem)

Reply:
The DatabaseAccount class doesn't contain the name value there, not sure that it's possible to get it